dbt/models/default/schema.yml

```diff
@@ -7,7 +7,7 @@ models:
     tests:
       # Unique by 14-digit PIN and year
       - unique_combination_of_columns:
-          name: unique_by_14_digit_pin_and_year
+          name: vw_pin_history_unique_by_14_digit_pin_and_year
```

Comment:
An example of what test output looks like before adding this namespacing, to motivate the change:

```
19:53:13  1 of 19 START test unique_by_14_digit_pin_and_year ................... [RUN]
```

There are a bunch of tests named `unique_by_14_digit_pin_and_year` in particular.

Reply:
Love this, good thinking!

**question (non-blocking)**: Is there no way to define the namespace in the test definition? If not, this totally works.